docs/source/configuration/index.md

```diff
@@ -1,13 +1,14 @@
 ---
 html_meta:
-  "description": ""
-  "property=og:description": ""
-  "property=og:title": ""
-  "keywords": ""
+  'description': ''
```

Comment:
I will fill in the meta attr shortly.

Reply:
Please do not change `'` to `"`. It's not necessary.

Also although I haven't tried it, the script that inserted all these placeholders might reinsert them if it is run again.

Reply:
Hm... my Editor to blame. It's configured as formatOnSave :(

Reply:
> Also although I haven't tried it, the script that inserted all these placeholders might reinsert them if it is run again.

I think it doesn't for doc files already having them: https://github.com/plone/documentation/blob/6-dev/docs/addMetaData.py#L33